api/osrd_infra/migrations/0015_operationalstudy_project_scenario_and_more.py

```diff
@@ -0,0 +1,131 @@
+# Generated by Django 4.1.5 on 2023-01-11 14:05
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
You have one big issue here. You need to manually create the ForeignKey and OneToOne fields. Ny default, these constraint are handle by django ORM and not Postgres engine. You should use the custom function `run_sql_add_foreign_key`. Some example are available in previous migrations.